Makefile

```diff
@@ -287,9 +286,16 @@ check-apply-migration:
 #############################
 
 # build devcontainer locally
+.PHONY: build-devcontainer
 build-devcontainer:
 	devcontainer build --workspace-folder=.github --push=false --image-name="ghcr.io/bucketeer-io/bucketeer-devcontainer:latest"
 
+.PHONY: push-devcontainer
+push-devcontainer:
```

Comment:
I think we shouldn't push our local dev container image to ghcr.
Because we only build our local machine arch image, it'll influence other user.

For local testing purpose, we don't need to push it to ghcr, it can use local image directly.

Reply:
I have removed it.